### PR TITLE
[710] Healthcheck smoke test

### PR DESCRIPTION
**Pull request by darokel**
### Context

-https://trello.com/c/kQtyhpFk/710-m-healthcheck-smoke-test

### Changes proposed in this pull request

- Create a new spec directory for smoke tests
- Setup a basic health check smoke test to assert response of system without stubs
- Setup a tag to run specs without smoke tests by default
- Run smoke tests in build workflow

### CODE REVIEW

**Comment by darokel** [comment]
[[M] Healthcheck smoke test](https://trello.com/c/kQtyhpFk/710-m-healthcheck-smoke-test)

**Review by felixtheflex — APPROVED** [review]

File: .github/workflows/smoke-test.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Smoke Tests ${{ github.event.inputs.environment }}
-        run: echo "Nothing to run yet 🚀"
+        run: bundle exec rspec --tag smoke
```
Comment (gpeng) [inline .github/workflows/smoke-test.yml:20]: Do we need to set the `RAILS_ENV` here to make sure `Settings.base_url` is returned correctly?

**Review by gpeng — COMMENTED** [review]
Did we check that the env is correctly set in the smoke

File: .github/workflows/smoke-test.yml (hunk 1 above)
Comment (darokel) [reply .github/workflows/smoke-test.yml]: have added in https://github.com/DFE-Digital/register-trainee-teachers/pull/372/commits/e8b8f6cc4127b370e70bd3f0385b7313a0d336c2

**Review by gpeng — APPROVED** [review]

**Pull request merged**